packages/core/src/lib/implementation/runner.ts

```diff
@@ -32,12 +34,13 @@ export type ValidatedRunnerResult = Omit<RunnerResult, 'audits'> & {
 
 export async function executeRunnerConfig(
   cfg: RunnerConfig,
+  config: Required<Pick<PersistConfig, 'outputDir'>>,
 ): Promise<RunnerResult> {
   const { args, command, outputFile, outputTransform } = cfg;
 
   const { duration, date } = await executeProcess({
     command,
-    args,
+    args: [...(args ?? []), ...objectToCliArgs(config)],
```

Comment:
Here could be potential for a more strict parsing. If we pass persist options directly to args it might cause unexpected issues.